### DISCUSSION TITLE
Public links to Github, docs and coffee

### DISCUSSION THREAD

**Discussion by ManuelLeiner** [opcodesio/log-viewer#249] [Ideas]
Hi there,

First of all, excellent package! I could have created an issue/bug report but let's be honest, the following is not actually something that is broken.

We might not be able to use this package due to the links to Github, the documentation and the coffee. I totally understand that you as the package owner deserve the recognition, but at the same time our customers will not be the target audience. The links won't even work when the log viewer is hosted _on-premises_ in a closed network.

I am happy to provide a PR to conditionally display/hide:

-  the following links:

<img width="270" alt="image" src="https://github.com/opcodesio/log-viewer/assets/8204574/e4b7b5a3-669a-4383-9228-bafe46e5bc5f">

- and this icon here:

<img width="173" alt="image" src="https://github.com/opcodesio/log-viewer/assets/8204574/bfa3b2fa-bd3e-4f4b-9b11-da57672c9534">

Is there a realistic chance of such a PR being merged? I saw that the link in the [bottom right corner](https://github.com/opcodesio/log-viewer/blob/main/src/Http/Controllers/IndexController.php#L22) is already configurable.

Keep up the great work and let me know if I can help. 🚀

**Comment by arukompas** [comment]
Hey!

I think it's fair to have these links on an open-source package that's free to use and is constantly maintained. It does help motivate me when someone appreciates the work and sends me a coffee 😀 

I think the solution to your situation is fairly easy - fork the repository. You can still have the latest features by pulling latest changes from this repo once in a while, and I think that's a small price to pay for you being able to remove the branding/links/coffee for your users.

Let me know what you think!

**Reply by ManuelLeiner** [reply]
This is more than fair 💯  We will discuss other solutions first, but good idea.

Appreciate the response! Cheers.
